src/org/zaproxy/zap/extension/ascan/ChallengeCallbackAPI.java

```diff
@@ -102,19 +102,17 @@ public void cleanExpiredCallbacks() {
      * @return a ZAP API URL to access the the challenge endpoint 
      */
     public String getCallbackUrl(String challenge) {
-        String callbackUrl = "http://" 
-                + Model.getSingleton().getOptionsParam().getProxyParam().getProxyIp() + ":" 
-                + Model.getSingleton().getOptionsParam().getProxyParam().getProxyPort() + "/" 
-                + getPrefix() + "/" 
-                + challenge;
-        
-        /* Key is not currently used for shorcuts... very interesting
+        /* Key is not currently used for shortcuts... very interesting
```

Comment:
This commented code could be removed as well.

Reply:
@thc202, the comment is removed as well.